.github/workflows/on-nextjs-release-publish.yml

```diff
@@ -7,6 +7,13 @@ on:
     # This is the event type defined by next.js upstream's `repository_dispatch` workflow dispatches.
     types: [nextjs-release-published]
   workflow_dispatch:
+    inputs:
+      version:
```

Comment:
Would you able to add some conditions in https://github.com/vercel/turbo/blob/b11becb48f75b1ef735ea99c7c84a0d6486e25af/.github/workflows/upload-nextjs-integration-test-results.yml#L56

to skip actual commit if this is triggered via manual dispatch? Currently we have some unexpected results uploaded https://github.com/vercel/turbo/tree/nextjs-integration-test-data/test-results without `RESULT_SUBPATH` substitution for the manual dispatch.